doc/ug/install_instructions/index.md

```diff
@@ -426,3 +426,15 @@ You then need to reload the udev rules:
 ```console
 $ sudo udevadm control --reload
 ```
+
+## Siemens Questa
+
+If using Siemens Questa, once a standard installation been completed the following additional steps are required:
+1. Add `QUESTA_HOME` as an environment variable which points to the Questa installation directory.
+2. Run `hw/dv/tools/questa/questa_initial_setup.sh`. This removes the `"# "` at the start of every transcript line to provide compatibility OpenTitan for log file error checking.
+
+As of Questa version 21.4 there are some code incompatibilities with the OpenTitan code-base:
+* It does not support associative arrays when using class objects as an index. Code related to this must be commented out.
+* Clocking and non clocking drive signals cannot be both used in tl_host_driver.sv. See issue 9514 for a temporary workaround for this.
+
+Support cases for both these issues have been raised with Siemens. Hopefully they will be fixed in a future release.
```

Comment:
```suggestion
As of Questa version 21.4 there are some code incompatibilities with the OpenTitan code-base.
See issue [#xxxx](link) for the list of issues and temporary workarounds. 
```

IMO the install guide is not the best place for listing these issues. I suggest instead opening an issue that tracks the list of problems faced, and link that bug here. 

Also, our guideline for markdown documents is one full sentence per line.